scripts/k8s/deployer.sh

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR="$(dirname "$0")"
+TMP_KIND="/tmp/kind/mayastor"
+TMP_KIND_CONFIG="$TMP_KIND/config.yaml"
+WORKERS=2
+DELAY="false"
+CORES=1
+POOL_SIZE="200MiB"
+DRY_RUN=
+KIND="kind"
+FALLOCATE="fallocate"
+KUBECTL="kubectl"
+DOCKER="docker"
+HUGE_PAGES=1800
+LABEL=
+INSTALL_ZFS="false"
+INSTALL_LVM="false"
+
+help() {
+  cat <<EOF
+Usage: $(basename "$0") [COMMAND] [OPTIONS]
+
+Options:
+  -h, --help                        Display this text.
+  --workers       <num>             The number of worker nodes (Default: $WORKERS).
+  --cores         <num>             How many cores to set to each io-engine (Default: $CORES).
+  --delay                           Enable developer delayed mode on the io-engine (Default: false).
+  --disk          <size>            Add disk of this size to each worker node.
+  --dry-run                         Don't do anything, just output steps.
+  --hugepages     <num>             Add <num> 2MiB hugepages (Default: $HUGE_PAGES).
+  --label                           Label worker nodes with the io-engine selector.
+  --install-zfs                     Install ZFS utilities.
```

Comment:
probably --setup-$engine might might be more appropriate as there might be other tasks other than installing utilities?